llvm/test/ThinLTO/AArch64/cgdata-merge-crash.ll

```diff
@@ -0,0 +1,6 @@
+; This test checks if the global merge func pass should not build module summary.
```

Comment:
Just a nitpick: should we move the test file into `llvm/test/CodeGen/AArch64/` or even `llvm/test/CodeGen/Generic/` since this is not a ThinLTO/backend-specific test?